modules/pmid/modules/pmid_populator/Readme.md

```diff
@@ -8,10 +8,10 @@ Populator plugin for PubMed, to create citation objects.
 
 This module requires the following modules/libraries:
 
-* [Islandora Scholar](https://github.com/Islandora/islandora_scholar)
-* [Islandora Importer](https://github.com/Islandora/islandora_importer)
-* [Pubmed Support Code](https://github.com/Islandora/islandora_scholar/modules/pmid)
+* [Islandora](https://github.com/Islandora/islandora)
 * [Islandora Populator](https://github.com/Islandora/islandora_populator)
+* [Islandora Scholar](https://github.com/Islandora/islandora_scholar)
+* [Islandora PMID](https://github.com/Islandora/islandora_scholar/modules/pmid)
```

Comment:
I think we need this link https://github.com/Islandora/islandora_scholar/pull/257/files#diff-c3d8b3537aaf1151bc66fb852fb44283R14 instead of https://github.com/Islandora/islandora_scholar/modules/pmid